src/types/time.ts

```diff
@@ -1,9 +1,10 @@
-import {STRules} from '../rules';
 import {StrongType, makeStrong} from '../strong-type';
 
+import {STRules} from '../rules';
+
 export type StrongTime = StrongType<string>;
 
-export function makeTime(fallback: string, initial?: string | null): StrongTime {
+export function makeTime(fallback: string, initial: string | null | undefined): StrongTime {
```

Comment:
- [ ] Change `| undefined` type on `initial` to use the optional operator instead `?`.